django_rest_generator/client.py

```diff
@@ -221,6 +221,7 @@ def _build_resource_object(self, resource_name: str, endpoints: dict):
         object_method_map = {
             "POST": [CreateableAPIResourceMixin],
             "GET": [ListableAPIResourceMixin, PaginationAPIResourceMixin],
+            "DELETE": [DeletableAPIResourceMixin],
```

Comment:
You cannot delete whole objects in DRF. This will be the equivalent of `curl -xDELETE http://localhost:8080/api/v2/skus/`